src/main/java/com/carrotcreative/cream/strategies/generic/CacheStrategy.java

```diff
@@ -6,14 +6,14 @@
 
 import java.io.Serializable;
 
-public interface CacheStrategy<T extends LoaderParams> {
+public interface CacheStrategy<T extends LoaderParams, C extends Serializable> {
```

Comment:
I am a little inclined towards letters, because they don't define what it means. So if `T` extends `LoaderParams` I know that T is (generic)something that extends `LoaderParams`. 

But if we Name it `Param`, it sounds more like a name of a Model rather than just being type. In dense code sometime it becomes confusing to notice if `Param` is a generic Type parameter or actually a model. 
So I am a little inclined towards Letters, but overall I can totally go both ways! :smile_cat: